.metadata

```diff
@@ -0,0 +1,157 @@
+# Copyright 2019 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+# ----------------------------------------------------------------------
+---
+# command line arguments
+params:
+  existing_version: 4.3.0
+  new_version: 4.4.0
+  wso2sp: "wso2sp-4.4.0"
+# files based configurations
+files:
+#Worker Configuration and Version changes
```

Comment:
Let's keep consistency in the comments. Start with a # and space and then start the comment with a simple letter.
Eg: `# sample comment`